package.json

```diff
@@ -106,7 +106,7 @@
     "@elastic/apm-rum-react": "^1.3.4",
     "@elastic/apm-synthtrace": "link:bazel-bin/packages/elastic-apm-synthtrace",
     "@elastic/charts": "45.1.1",
-    "@elastic/datemath": "link:bazel-bin/packages/elastic-datemath",
+    "@elastic/datemath": "5.0.3",
```

Comment:
Is this still needed?

Reply:
It is due to `@elastic/eui` which declares a `peerDependency` on it instead of a `dependency`. If they change that we can remove this.